lib/peeppresenter.rb

```diff
@@ -0,0 +1,32 @@
+require './lib/timeago'
+class PeepPresenter
+  def self.run(peep)
+    PeepPresenter.new(TimeAgo).run(peep)
+  end
+
+  def initialize(time_ago_klass)
+    @time_ago_klass = time_ago_klass
+  end
+
+  def run(peep)
+    present(peep)
+  end
+  private
+
+  def present(peep)
+    "'<strong>#{body_text(peep)}</strong>' <sub>#{time_posted(peep)} ago from #{name_of_peeper(peep)}</sub>"
```

Comment:
Metrics/LineLength: Line is too long. [108/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)